src/platform/Linux/NetworkCommissioningDriver.h

```diff
@@ -147,11 +150,14 @@ class LinuxWiFiDriver final : public WiFiDriver
         Platform::SharedPtr<Crypto::P256Keypair> clientIdentityKeypair;
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFI_PDC
     };
-
-    WiFiNetwork mSavedNetwork;
-    WiFiNetwork mStagingNetwork;
+    uint8_t currentNetId          = 0;
+    uint8_t connectedNetworkIndex = 0;
+    WiFiNetwork mSavedNetwork[kMaxNetworks];
```

Comment:
Please update naming of these things to make it clear that there are multiple.

Reply:
Hi, Thank you for the feedback. Will update the PR with documentation and change the naming to the correct naming convention. There is still a bit of work remaining for this around storing the WiFi information in persistent storage. Will open a PR (not a draft) for review with all the changes in.